portfolio/src/main/webapp/test.js

```diff
@@ -0,0 +1,70 @@
+const one = document.querySelector('#box-area');
```

Comment:
Would recommend improving the naming in this file, I'm not sure what "test.js", "one", "...zeroVars", etc are referring to

Reply:
When I take a step back I definitely see that these names make no sense, re-worked the naming inside of the file to make variables actually readable

Reply:
The update looks great to me! Thanks Jamari :)